sections_flask-jwt-extended/app.py

```diff
@@ -0,0 +1,107 @@
+from flask import Flask, jsonify
+from flask_restful import Api
+from flask_jwt_extended import JWTManager
+
+from db import db
+from resources.user import UserRegister, UserLogin, TokenRefresh
+from resources.item import Item, ItemList
+from resources.store import Store, StoreList
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///data.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+api = Api(app)
+db.init_app(app)
+
+"""
+JWT related configurations began. The following functions includes:
+1) add claims to each jwt
+2) customize the token expired error message 
+"""
+app.config['JWT_SECRET_KEY'] = 'jose'   # we can also use app.secret like before, Flask-JWT-Extended can recognize both
+app.config['JWT_BLACKLIST_ENABLED'] = True  # enable blacklist feature
+app.config['JWT_BLACKLIST_TOKEN_CHECKS'] = ['access', 'refresh']  # allow blacklisting for access and refresh tokens
+jwt = JWTManager(app)
+
+"""
+`claims` are data we choose to attached to each jwt payload
+and for each jwt protected endpoint, we can retrieve these claims via `get_jwt_claims()`
+one possible use case for claims are access level control, which is shown below
+"""
+
+
+@jwt.user_claims_loader
+def add_claims_to_jwt(identity):
+    if identity == 1:   # instead of hard-coding, we can read from a config file to get a list of admins instead
+        return {'isAdmin': True}
+    return {'isAdmin': False}
```

Comment:
Can we stick to snake_case for consistency? `is_admin` instead

Reply:
My bad, will update it now.